packages/aws-cdk-lib/aws-route53/lib/alias-record-target.ts

```diff
@@ -25,4 +25,11 @@ export interface AliasRecordTargetConfig {
    * DNS name of the target
    */
   readonly dnsName: string;
+
+  /**
+   * Evaluate the target health
+   *
+   * @default - undefined
```

Comment:
The default value is `undefined`, but could you describe what settings are applied when it is `undefined`?

Reply:
hi @badmintoncryer , I've changed this to more descriptive information about the default value of this property. Could you please check and let me know what do you think?